doc/source/user_guide/index.rst

```diff
@@ -1,12 +1,10 @@
 .. _ref_user_guide:
 
 ==========
-User Guide
+User guide
 ==========
-This page introduces the PyFluent library for anyone who wants to
-import its Python modules and start to develop Python code to control
-and monitor Ansys Fluent. 
-
+Anyone who wants to use PyFluent can import its Python modules develop
+Python code to control and monitor Ansys Fluent. 
```

Comment:
Just noticed that you changed the reference it in a few other places from `Ansys Fluent` to `Fluent`.  I think we should be using the former no?

Reply:
@dnwillia-work All other PyAnsys libraries use Ansys prior to the name only on initial reference in a README or the doc landing page. Unless their is a legal requirement that Ansys always appear in your name? If so, I could do a global replacement. The doc currently probably had the Ansys prior to Fluent about 50% of the time.

Reply:
Yeah, I don't know what's best, there's no legal requirements that I know of so it would just be a convention.  I'm kind of agnostic and could go either way.  Maybe we just say `Ansys Fluent` in the README and use `Fluent` everywhere else?

Reply:
I completely agree, let's keep it simple.